Cutelyst/Plugins/Utils/Validator/validator.cpp

```diff
@@ -199,9 +203,9 @@ void Validator::fillStash(Context *c)
         c->setStash(QStringLiteral("validationErrorStrings"), validationErrorStrings);
         c->setStash(QStringLiteral("validationErrorFields"), validationErrorFields);
 
-        if (!d->params.isEmpty()) {
-            QMap<QString,QString>::const_iterator i = d->params.constBegin();
-            while (i != d->params.constEnd()) {
+        if (!params.isEmpty()) {
```

Comment:
It _can_ be empty, for example if there was no input data and validation failed because some values are required.